config/config.go

```diff
@@ -260,14 +260,18 @@ func (c *Config) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	// Do global overrides and validate unique names.
 	jobNames := map[string]struct{}{}
 	for _, scfg := range c.ScrapeConfigs {
+		if scfg.ScrapeTimeout > scfg.ScrapeInterval {
```

Comment:
You want to remove the scrape timing options from scrape configs entirely then?

A simpler option in general would be to silently set timeouts to `min(scrape_interval, scrape_timeout)` everywhere.